fe1-web/src/resources/strings.ts

```diff
@@ -291,6 +291,7 @@ namespace STRINGS {
     'The Roll Call is currently open and you as the organizer should start adding attendees by scanning their PoP tokens.';
   export const roll_call_open_attendee =
     'The Roll Call is currently open and you as an attendee should let the organizer scan your PoP token encoded in the QR Code below.';
+  export const roll_call_QRcode_text = 'To be scanned';
```

Comment:
I think this text is a bit too general, it would be nice to have something that indicates the purpose of the qrcode. It is clear that it should be scanned but what for?

Also it would be nice to stick to the naming conventions, i.e. camel case except in this file because of legacy reasons. At some point we can make it all camel case here too though😁